config/etfuturum/world.cfg

```diff
@@ -0,0 +1,192 @@
+    B:amethystDimensionBlacklistAsWhitelist=false
+
+    # Max Y level amethyst geodes should attempt to generate at [range: 6 ~ 245, default: 46]
+    I:amethystMaxY=46
+
+    # Use a namespaced ID, + optionally meta (max 15) to choose the block that makes up the middle layer of amethyst geodes.
+    # If the chosen block does not exist then amethyst geodes will not generate. [default: etfuturum:calcite]
+    S:amethystMiddleBlockID=etfuturum:calcite
+
+    # Use a namespaced ID, + optionally meta (max 15) to choose the block that makes up the outer layer of amethyst geodes.
+    # The outer layer was formerly "etfuturum:tuff" before it was changed in later 1.17 snapshots. Netherlicious smooth basalt is "netherlicious:BasaltBricks:6"
+    # If the chosen block does not exist then amethyst geodes will not generate. [default: etfuturum:smooth_basalt]
+    S:amethystOuterBlockID=etfuturum:smooth_basalt
+
+    # How rare should amethyst geodes be? 1/x chance per chunk, 1 means a geode attempts to appear every chunk [range: 1 ~ 127, default: 53]
+    I:amethystRarity=53
+
+    # 0 = Budding amethyst cannot be obtained at all even with silk touch. When using this option, attempting to push them using a piston will break it.
+    # 1 = Budding amethyst will drop if you use a silk touch pickaxe.
+    # 2 = Budding amethyst does not need silk touch, just a pickaxe. [range: 0 ~ 2, default: 0]
+    I:buddingAmethystMode=0
+
+    # How rare should cherry trees be? 1/x chance per chunk, 1 means a tree attempts to appear every chunk. 0 = no cherry trees. They will spawn in mountain-type biomes. [range: 0 ~ 127, default: 72]
+    I:cherryTreeRarity=72
+
+    # Max vein size for copper ore blocks in a cluster [range: 0 ~ 64, default: 8]
+    I:copperClusterSize=8
+
+    # The max vein size for the second, typically bigger debris veins, which generate from Y 8 to 22 [range: 0 ~ 64, default: 3]
+    I:debrisMax=3
+
+    # If deepslateGenerationMode is set to 1, this value is used to determine how big the clusters are. Otherwise this value is unused. [range: 0 ~ 128, default: 64]
+    I:deepslateClusterSize=64
+
+    # If 0, deepslate replaces all stone below the specified value, with a shattering effect near the top similar to bedrock. If 1, it generates in clusters using the deepslate cluster settings. -1 disables Et Futurum deepslate generation entirely. [range: -1 ~ 1, default: 0]
+    I:deepslateGenerationMode=0
+
+    # The dimensions the deepslate layer (deepslate generation mode 0) should not spawn in. Does nothing if other deepslate generation modes are used.
+    I:deepslateLayerDimensionBlacklist <
+        -1
+        1
+     >
+
+    # Treat the deepslate layer dimension blacklist as a whitelist instead, so it will ONLY generate in those dimensions, instead of excluding those dimensions from generation. [default: false]
+    B:deepslateLayerDimensionBlacklistAsWhitelist=false
```

Comment:
whitelist for sure

Reply:
Done. And added the overworld as the only dimension in the allowlist.